lib/perl/Genome/Db/Ensembl/Command/Run/Base.pm

```diff
@@ -669,4 +692,16 @@ sub cache {
     return $cache_result;
 }
 
+sub _result_users {
+    my $self = shift;
+
+    my $analysis_build_anp = $self->analysis_build?
+        $self->analysis_build->model->analysis_projects : undef;
+
+    return {
+        requestor => ($self->analysis_build // $self->analysis_process),
+        sponsor => ($self->analysis_project // $analysis_build_anp // Genome::Sys->current_user),
```

Comment:
I've noticed several places using `//`?  Are you intentionally suggesting that zero values exist and are okay?